packages/website/docs/entities/ui-schema.md

```diff
@@ -31,14 +31,15 @@ for rendering. To switch layouts, [replacing a component repository](/docs/entit
 | `compoennts[name: string] [pointer: string].name`    | `string`                 |               | The name of the component to use at the above location, which has to be available in the above component repository                                                  |
 | `compoennts[name: string] [pointer: string].options` | `any`                    |               | Options to pass to the component at the above location                                                                                                               |
 
-## the `components` UI hint
```

Comment:
@yoavaa take a look plz, I removed `UI hint` wording, as it might be confusing. Because we don't use it in types, for types `UIHints` is the `UISchema.hints` property only.